src/runtime-rs/crates/hypervisor/src/device/blk_dev_manager.rs

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+use std::{collections::HashMap, str, sync::Arc};
+
+use anyhow::{anyhow, Context, Result};
+use async_trait::async_trait;
+use tokio::sync::Mutex;
+
+use crate::{
+    device_manager::{
+        get_host_path, KATA_BLK_DEV_TYPE, KATA_MMIO_BLK_DEV_TYPE, VIRTIO_BLOCK_MMIO,
+        VIRTIO_BLOCK_PCI,
+    },
+    device_type::{BlockDevice, Device, DeviceArgument, GenericConfig},
+    DeviceManagerInner, Hypervisor,
+};
+use kata_sys_util::rand;
+
+// Device manager will manage the lifecycle of sandbox device
```

Comment:
The comment does not match the `BlockDeviceManager`.

Reply:
fixed